### ISSUE TITLE
how to select db redis?

### ISSUE THREAD

**Issue by mvtcode** [socketio/socket.io-redis#161]
Default this module select db redis 0. But I want select db not equal 0, exam:

var io = require('socket.io')(3000);
var redis = require('socket.io-redis');
io.adapter(redis({ host: 'localhost', port: 6379, **db: 15** }));

How to be so?

**Comment by darrachequesne** [comment]
Currently, you'll have to provide your own pub / sub clients:
```js
var redis = require('redis').createClient;
var adapter = require('socket.io-redis');
var pub = redis(port, host, { auth_pass: "pwd", db: 15 });
var sub = redis(port, host, { auth_pass: "pwd", db: 15 });
io.adapter(adapter({ pubClient: pub, subClient: sub }));
```
`options` argument will be forwarded to `redis` in the upcoming release:
```js
var io = require('socket.io')(3000);
var redis = require('socket.io-redis');
io.adapter(redis({ host: 'localhost', port: 6379, db: 15 }));
```

Related: https://github.com/socketio/socket.io-redis/pull/157

**Comment by darrachequesne** [comment]
Released as `3.0.0`.
